### ISSUE TITLE
[New build - DEFAULT] 20w20d: Hotfixes and invalid link warning.

### ISSUE THREAD

**Issue by tsunamayo** [tsunamayo/Starship-EVO#2077]
Some hotfixes regarding turrets and more warning message to prevent invalid link operations.

Changes:
 - Added a warning message when trying to connect invalid brick to yoke.
 - Added a warning message when no weapon is assigned to a turret.

Bug Fixes:
#2064 Wrong Push Tool icon in help message.
#2065 Turret clip in parent.
#2071 Can link Camera to Yoke.
#2076 Query gate does not accept decimal.
#2074 #2072 Turret tracking issue

**Comment by tsunamayo** [comment]
Build is live. Thanks!

**Comment by ProPeach** [comment]
Thanks for the fixes!
I'm not sure if you saw it or not, but I went through the tutorial and posted some potential errors in #2023

**Comment by tsunamayo** [comment]
Yes I saw, thanks a lot for taking the time to do this - I will do all the change!
